src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px
+    rgba(0, 0, 0, 0.05);
+  background-color: #fff;
+  
+  &__logo {
+    position: absolute;
+    top: 10px;
+    bottom: 10px;
+  }
+  
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+  
+  &__item {
+    display: flex;
+  }
+  
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+  
+  &__link {
+    @include font-parameters ($fontFamily, 500, normal);
+    font-size: 12px;
+    text-decoration: none;
+    text-transform: uppercase;
```

Comment:
According to changes in html it would be elements of `nav` block. Move this code in `nav.scss` file.